platform-emf/src/main/java/com/softicar/platform/emf/attribute/field/foreign/entity/input/edit/EmfEntityInputEditButton.java

```diff
@@ -39,18 +41,20 @@ private boolean isEditAllowed(E tableRow) {
 			.test(tableRow, CurrentBasicUser.get());
 	}
 
-	private class FormPopup extends EmfFormPopup<E> {
+	private class FormPopup extends EmfFormPopup<E> implements IDomRefreshBusListener {
```

Comment:
hmm, while this works, I think the input itself should rather listen to the change...but this might only be feasible after the refactoring of the auto-complete input, b/c there are other possibilities for a user to change the selected value of the auto-complete, this fix only solves one case...we need to talk with Alex

Reply:
created PLAT-1106